docker-compose.yml

```diff
@@ -155,35 +217,6 @@ services:
             ip link set blue up && \
             ip link add green type vrf table 1001 && \
             ip link set green up && \
-            ip link add yellow type vrf table 1002 && \
-            ip link set yellow up && \
-
-            ip link add br100 type bridge && \
-            ip link set br100 master blue addrgenmode none && \
-            ip link set br100 addr aa:bb:cc:00:00:02 && \
-            ip link add vni100 type vxlan local 10.0.0.4 dstport 4789 id 100 nolearning && \
-            ip link set vni100 master br100 addrgenmode none && \
-            ip link set vni100 type bridge_slave neigh_suppress on learning off && \
-            ip link set vni100 up && \
-            ip link set br100 up && \
-
-            ip link add br101 type bridge && \
-            ip link set br101 master green addrgenmode none && \
-            ip link set br101 addr aa:bb:cc:00:00:03 && \
-            ip link add vni101 type vxlan local 10.0.0.4 dstport 4789 id 101 nolearning && \
-            ip link set vni101 master br101 addrgenmode none && \
-            ip link set vni101 type bridge_slave neigh_suppress on learning off && \
-            ip link set vni101 up && \
-            ip link set br101 up && \
-
-            ip link add br102 type bridge && \
-            ip link set br102 master yellow addrgenmode none && \
-            ip link set br102 addr aa:bb:cc:00:00:04 && \
-            ip link add vni102 type vxlan local 10.0.0.4 dstport 4789 id 102 nolearning && \
-            ip link set vni102 master br102 addrgenmode none && \
-            ip link set vni102 type bridge_slave neigh_suppress on learning off && \
-            ip link set vni102 up && \
-            ip link set br102 up && \
```

Comment:
I believe new updated diagramms are needed so I can understand what is the networking configuration on other leafs beside leaf1 so I can understand what we are testing and what we trying to ping. This way I can judge the changes effectively.